Updates.md

```diff
@@ -11,26 +11,29 @@ The Web Application Manifest for an Isolated Web App will include two new fields
 *   the `version` field allows the app publisher to identify a Web Bundle as containing a particular version of the application
 *   the `update_manifest_url` field tells the user agent where to look for updated versions of the application
 
-The resource served at the `update_manifest_url` is a JSON document listing versions of the application that are available and the URL from which the Web Bundle containing that version can be downloaded.
+The resource served at the `update_manifest_url` is a JSON document listing versions of the application that are available and the URL from which the Web Bundle containing that version can be downloaded. It also specifies which _Update Channels_ each version is part of.
 
 In order to check for updates the user agent follows the following steps,
 
 1. Fetch the resource from `update_manifest_url` and parse it as a JSON document.
 2. Let _selectedVersion_ and _selectedSrc_ be initialized to `null`.
-3. For each object in the document’s `versions` entry,
+3. Let _updateChannel_ be the identifier of the Update Channel the installed app is subscribed to.
+4. For each object in the document’s `versions` entry,
     1. Let _version_ be the object’s `version` field, skipping this item if the field does not exist.
     2. Let _src_ be the object’s `src` field, skipping this item if the field does not exist.
-    3. If _version_ is not a valid version identifier or _src_ is not a valid URL, skip this item.
-    4. If _selectedVersion_ is `null` or _selectedVersion_ is less than or equal to _version_, set _selectedVersion_ to _version_ and _selectedSrc_ to _src_. 
-4. If _selectedVersion_ is `null`, abort these steps.
-5. If _selectedVersion_ is less than or equal to the currently installed version, abort these steps.
-6. Fetch the resource from _selectedSrc_.
-7. If the resource is not a valid Signed Web Bundle, abort these steps.
-8. If the Web Bundle ID of the resource is not equal to that of the currently installed app, abort these steps.
-9. Load `/manifest.webmanifest` from the Web Bundle and parse it as a Web Application Manifest.
-10. If the resource is not found or fails to parse, abort these steps.
-11. If the `version` field in the manifest from the Web Bundle is not equal to _selectedVersion_, abort these steps.
-12. Replace the Web Bundle for the current installed app with the downloaded resource.
+    3. Let _channels_ be the object’s `channels` field, defaulting to a list with a single element named `default` (`["default"]`) if the field does not exist.
+    4. If _version_ is not a valid version identifier or _src_ is not a valid URL or _channels_ is not a list of valid channel identifiers, skip this item.
```

Comment:
If any arbitrary string is not a "valid channel identifier" we need to define what the restrictions are.

Reply:
Done - added a sentence about that further down ("An Update Channel is identified by its ID, which MUST be a non-empty string.").